README.md

```diff
@@ -35,8 +38,23 @@ A project is configured using a simple json like that
 }
 ```
 
+- For `freelancer` mode: 
+  * not implemented
+
+- For `team` mode:
+  * not implemented
+
 After to configure, the budget is ready to build
 
+# Args
+
+The possible arguments for allspark are: 
+  - `-v` or `--version`: Prints the project version.
+  - `-c` or `--config`: Specifies the config file path. It's required.
+  - `-o` or `--output`: Specifies the output directory to write the budget.html file. Defaults to `./`.
+  - `-m` or `--mode`: Specify wich mode of budget you want. The config file should be specified in this mode. Defaults to `project`.
+  - `-s` or `--shine`: Tells allspark to enligth you with all the information of whats is being done behind the scenes (verbose). Defaults to `false`.
```

Comment:
Good idea but I think that is more semantic to use the word `-v --verbose` is the standard in a lot of cli tools